bundles/org.eclipse.equinox.common/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.equinox.common; singleton:=true
-Bundle-Version: 3.16.0.qualifier
+Bundle-Version: 3.16.100.qualifier
```

Comment:
Why that? We only change version once in a release, if minor segment was already touched.

Reply:
Yes, but the increment to 3.16.0 was already done for the 2022-03 release, which this change is for 2022-06.

Because the new method was added late in the development process Tom recommended to postpone the deprecation to 2022-06 to give users in the platform the time to react to the deprecation.

Reply:
sorry, I assumed the change were just created.

Reply:
No problem. I always appreciate if somebody reviews my changes :)